testcases/kernel/controllers/memcg/stress/memcg_process_stress.c

```diff
@@ -92,13 +92,18 @@ int main(int argc, char *argv[])
 		interval = 1;
 
 	/* TODO (garrcoop): add error handling. */
-	memset(&sigint_action, 0, sizeof(sigint_action));
+	/* TODO(mw): DRY it */
```

Comment:
Did you mean "don't repeat yourself"? Given the minimal numbers of change, I don't think adding either a macro or static function that does the two calls really is worth the merit of the additional complexity.